be/src/exec/pipeline/source_operator.h

```diff
@@ -39,11 +39,10 @@ class SourceOperatorFactory : public OperatorFactory {
     // OlapScanOperator needs to do so.
     virtual bool with_morsels() const { return false; }
     // Set the DOP(degree of parallelism) of the SourceOperator, SourceOperator's DOP determine the Pipeline's DOP.
-    virtual void set_degree_of_parallelism(size_t degree_of_parallelism) {
-        this->_degree_of_parallelism = degree_of_parallelism;
-    }
+    void set_degree_of_parallelism(size_t degree_of_parallelism) { _degree_of_parallelism = degree_of_parallelism; }
```

Comment:
how about unify the word `dop` instead of such a verbose phrase ?

Reply:
Good advice, it's too tedious.